README.md

```diff
@@ -81,7 +81,8 @@ If you want Papermill rendering to stop on a Spark error, edit the `~/.sparkmagi
 ```json
 {
     "spark_statement_errors_are_fatal": true,
```

Comment:
I believe this particular config option was added specifically for the Papermill use case, but doesn't actually work—maybe it should be ripped out?